guide/getting-started/software.md

```diff
@@ -152,10 +148,14 @@ Decentralized exchanges fall in between these categories, focusing on occasional
    layout = "float-right-desktop"
 %}
 
-A [node]({{ '/guide/glossary/node/' | relative_url }}) is simply a computer running a Bitcoin software. Nodes verify transaction data on the Bitcoin blockchain and ensure the Bitcoin rules are being followed by network participants. Some wallets have built-in nodes, and some node software comes packaged with wallet functionality. However, the minimum implementation of a node is syncing data and allowing other software a user has to access it.
+A Bitcoin [node]({{ '/guide/glossary/node/' | relative_url }}) is simply a computer running software, like [Bitcoin Core]({{ '/guide/glossary/#bitcoin-core-client'}}) connecting in to the Bitcoin network. Nodes verify transaction data on the Bitcoin blockchain and ensure the Bitcoin rules are being followed by network participants. Some wallets have built-in nodes, and some node software comes packaged with wallet functionality. However, the minimum implementation of a Bitcoin node is syncing data and allowing other software a user has to access it.
```

Comment:
```suggestion
Bitcoin [node]({{ '/guide/glossary/node/' | relative_url }}) software is simply software that connects to and participates in the Bitcoin network. [Full nodes]({{ '/guide/glossary/node/#full-nodes/' | relative_url }}) verify transaction data on the Bitcoin blockchain and ensure the Bitcoin rules are being followed by network participants. A [light node]({{ '/guide/glossary/node/#light-nodes/' | relative_url }}) connects to the Bitcoin network but does not verify or enforce the rules of the Bitcoin network. Some wallet software comes with built-in full node capabilities, such as [Bitcoin Core]({{ '/guide/glossary/#bitcoin-core-client/' | relative_url }}), but most wallet software act as light nodes. 
```

Made it more software contextual / technically accurate rather than simply describing what a node is.

Reply:
I removed some repeating words and committing.